### PR TITLE
32 plants discover page

### PR DESCRIPTION
**Pull request by ajkrumholz**
### User Story Implemented
#32 Plant Discover Page

### Describe Features Added/Changed
Add a plant discover page accessible here:
/gardens/:garden_id/plots/:plot_id/plants/discover
Displays a table of all plants native to the garden's state with an option to add them to the garden.

### Describe Tests Added/Changed
Happy paths tested with stubbed json responses

### Checklist before requesting a review
- [x] Added features have thorough tests written for their functionality?
- [x] All Tests Passing?

### CODE REVIEW

**Review by musselmanth — DISMISSED** [review]
Part of this user story is the creation of plot plants when the button is pushed. AJ is going to implement that and repush before we review and merge.

File: app/facades/garden_facade.rb [hunk 1]
```diff
@@ -16,7 +16,7 @@ def self.get_garden_plots(garden_id)    #get_garden_plots_endpoint
 
   def self.get_user_gardens(user_id)    #get_user_gardens_endpoint
     json = DatabaseService.get_user_gardens_endpoint(user_id)
-    json[:data].map { |garden| Garden.new(garden[:attributes], garden[:id])}
+    json[:data].map { |garden| GardenPoro.new(garden[:attributes], garden[:id])}
```
Comment (sjmann2) [inline app/facades/garden_facade.rb:19]: This is just Garden.new now

Comment (sjmann2) [inline app/facades/garden_facade.rb:19]: This is just Garden.new now

File: app/views/plants/index.html.erb [hunk 2]
```diff
@@ -1 +1,23 @@
...
+            <%= f.check_box "plant_ids", { multiple: true }, plant.id, nil %>
+          </td>
+          <td><%= plant.common_name.titleize %></td>
+          <td><%= plant.scientific_name.titleize %></td>
+          <td><%= plant.suitable_for_hz %></td>
+        </tr>
+      <% end %>
+    </table>
+  <% end %>
+
```
Comment (sjmann2) [inline app/views/plants/index.html.erb:23]: great implementation of a table love that

**Review by sjmann2 — CHANGES_REQUESTED** [review]

**Review by sjmann2 — APPROVED** [review]

**Pull request merged**